source/client.js

```diff
@@ -13,7 +15,11 @@ const pageDetails = parse(window.location.search)
 
 render(
   <div>
```

Comment:
Oh I see... yeah, better to pull this out into a `Root` or `App` component. You can `renderToStaticMarkup` it during build. Will show tomorrow if you like?